src/interface.jl

```diff
@@ -38,26 +39,32 @@ end
 ######################
 
 "Abstract type for a `Bijector`."
-abstract type Bijector end
+abstract type Bijector{N} end
```

Comment:
Not compulsory but would be nice to rename this to `AbstractBijector`.

Reply:
Suppose `N` refers to the dimension, maybe rename it to `D`?

Reply:
Sure to both!

Reply:
Actually, I now remember why I used `N` instead of `D`. 

You're often going to see `D` used as a `D<:Distribution` _and_ `N` is also used to denote the same thing in `AbstractArray{T, N}`, so this is why I decided to do that. And I think I'd prefer sticking with `N`